code/modules/emotes/definitions/slimes.dm

```diff
@@ -2,11 +2,12 @@
 	key = "nomood"
 	var/mood
 
-/decl/emote/slime/do_extra(var/mob/living/simple_mob/slime/user)
+/decl/emote/slime/do_extra(mob/user, atom/target)
```

Comment:
I have a preference worked into me for parameters to be typed for their expected input because it saves specifying the path twice and creating an extra reference, but I only add this as a comment